bionic/flow.py

```diff
@@ -17,7 +17,7 @@
 
 # A bit annoying that we have to rename this when we import it.
 from . import protocols as protos
-from .cache import LocalStore, GcsCloudStore, PersistentCache
+from .persistence import LocalStore, GcsCloudStore, PersistentCache
```

Comment:
super nit: reorder imports. I think we talked about adding a hook for this, not sure where we landed with it.

Reply:
Thanks, fixed!